src/eventDispatchers/shared/customCallbackHandler.js

```diff
@@ -15,10 +15,11 @@ export default function (handlerType, customFunction, evt) {
 
   if (handlerType === 'touch') {
     tools = tools.filter(
-      (tool) => tool.options.touchEnable || tool.options.touchEnable === undefined
-    );  
+      (tool) =>
```

Comment:
Maybe we shouldn't treat `undefined` as touch enabled? There was a previous bit of code PR'd from someone on @diego0020's team, I believe, that had it working this way. I was mostly updating the config option's name here.

I feel like setting `touch` active should be deliberate? We don't default `mouseButtonMask` to 1, so we probably shouldn't default `touch` to active either.

> We could have some default _configuration in BaseTool, and then Object.assign(this._configuration, confguration), where configuration is the argument passed to the constructor of BaseTool.

I think that makes sense for initialization.

Reply:
`isTouchActive` is weird. It's primarily meant to make activating a tool more clear when it works for both `touch` and `mouse`. Kind of a... YOU ARE activating this for mouse, but "are you sure?" you want it to become the active `touch` tool as well.

If that's what we're using it for, then it doesn't make sense on any `touch` tool that doesn't also support `mouse`. For example, a `doubleTap` or gesture based tool would not be activated ONLY for mouse, so it's safe to assume we're activating it for it's specific `touch` scenario.

It's almost like we need a `touchMask` that indicates "off", single finger, 2-finger-touch, 3-finger-touch, specific gesture, etc. But with the caveat that some `touch` tools can only support a subset of that mask.

Confusing stuff. As of right now, I can't think of an elegant solution -- what we have "works", but I imagine this will be something that gets polished up in a later major version.